src/components/organisms/pupil/OakLessonLayout/OakLessonLayout.tsx

```diff
@@ -64,45 +65,82 @@ export const OakLessonLayout = ({
       <OakFlex
         $flexDirection="column"
         $flexGrow={1}
-        $background={[mobileContentBackgroundColor, contentBackgroundColor]}
-        $btr={[null, "border-radius-xl"]}
-        $bt={[null, "border-solid-xl"]}
-        $bh={[null, "border-solid-xl"]}
-        $borderColor={[null, contentBorderColor]}
+        $background={[mobileContentBackgroundColor, "transparent"]}
         $maxWidth="all-spacing-24"
         $minHeight="100%"
         $mh="auto"
-        $pt={["inner-padding-none", "inner-padding-m"]}
-        $gap={["space-between-l", "space-between-xl"]}
+        $position={"relative"}
       >
-        {topNavSlot && (
-          <OakBox
-            $pv="inner-padding-l"
-            $pl={["inner-padding-m", "inner-padding-xs"]}
-            $pr={["inner-padding-m", "inner-padding-none"]}
-            $mr={["space-between-none", "space-between-l"]}
-            $background={["bg-primary", "transparent"]}
+        <OakBox $display={["none", "block"]}>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="100%"
+            height="100%"
+            viewBox="0 0 1308 943"
+            fill="none"
+            style={{
+              position: "absolute",
+              top: 0,
+              left: 0,
+              right: 0,
+              bottom: 0,
+              zIndex: 0,
+            }}
+            preserveAspectRatio="none"
           >
-            {topNavSlot}
-          </OakBox>
-        )}
-        <OakFlex $flexGrow={1} $flexDirection="column">
-          {children}
+            <mask id="wobbly_mask" fill="white">
```

Comment:
I know I'm guitly of it too, but I'm not very fond of us inlining SVGs like this. We need to set up a folder for SVG files and then import them.